src/operations/operation.ts

```diff
@@ -33,8 +34,9 @@ export class OperationBase<T extends OperationOptions = OperationOptions> {
   ns!: MongoDBNamespace;
   cmd!: Document;
 
-  readPreference?: ReadPreference;
+  readPreference!: ReadPreference;
```

Comment:
why the `!`?

Reply:
In cursor's get readPreference it can't return undefined for a readPref, I moved the assertion there instead.